### ISSUE TITLE
A question regarding the data setting

### ISSUE THREAD

**Issue by BeileiCui** [YuemingJin/STswinCL#3]
Hi. In seg18/dataset/Endovis2018_new.py the 15 training videos are seperated into 12 train set and 3 validation set. However, during training the network were trained with all the 15 train set and make the validation on the 4 test set. May I ask the reason for doing this?

**Comment by YuemingJin** [comment]
Hi, thanks for your interest in our work. In the real training procedure, we exactly follow the evaluation protocol of the Challenge - train by 15 videos, and test on 4 videos. We combine 12/3 to 15 train set in Line 51. You could ignore this separation, which is just used for our initial attempt. Sorry for the confusion.
